docs/content/en/docs/getting-started/production-environment/vsphere-getstarted.md

```diff
@@ -177,40 +178,54 @@ Follow these steps if you want to use your initial cluster to create and manage
    export EKSA_LICENSE='my-license-here'
    ```
 
-1. Create a workload cluster
+1. Create a workload cluster in one of the following ways:
+   
+   * **GitOps**: Recommended for more permanent cluster configurations. See [Manage cluster with GitOps]({{< relref "/docs/tasks/cluster/cluster-flux" >}}) for details.
```

Comment:
should to add here that uses need to add the `kustomization.yaml` in the same folder as the workload cluster config file in order to trigger the creation?

Also, might me worth updating `/docs/tasks/cluster/cluster-flux` mentioning that GitOps now supports worklooad cluster creation and deletion? Just because there is a section specifying what fields can be updated and which can't.

Reply:
Yeah. We should do that. I'll give it a shot.